package.json

```diff
@@ -22,7 +22,7 @@
   "devDependencies": {
     "babel-eslint": "^6.1.0",
     "snazzy": "^4.0.0",
-    "standard": "^7.1.2"
+    "standard": "^8.0.0"
```

Comment:
Understand. I was trying to fix the Appveyor build, but it seems to break Travis in the process. I'll roll it back.